src/components/Lightbox.tsx

```diff
@@ -64,30 +59,54 @@ function Lightbox({
     index = 0,
     activeIndex = 0,
     hasSiblingCarouselItems = false,
-    zoomRange = defaultZoomRange,
+    zoomRange = DEFAULT_ZOOM_RANGE,
 }: LightboxProps) {
     const StyleUtils = useStyleUtils();
 
-    const [containerSize, setContainerSize] = useState<ContentSize>({width: 0, height: 0});
-    const isContainerLoaded = containerSize.width !== 0 && containerSize.height !== 0;
+    const [canvasSize, setCanvasSize] = useState<ContentSize>({width: 0, height: 0});
```

Comment:
```suggestion
    const [canvasSize, setCanvasSize] = useState<CanvasSize | undefined>(undefined);
```

Let's init this with [undefined](url) so we can simplify the `isCanvasLoaded` and also to be consist with `isContentLoaded`